docs/automated-process.md

```diff
@@ -60,13 +60,28 @@ release process. The automated processes uses GitHub Actions to automate the
    branches down to the ansible-core version that is included in the Ansible
    release.
 
-5. Announce the release on the Forum, Matrix, and the mailing list by running
-   the following command in the `${MAJOR_VERSION}` directory of the
-   `ansible-build-data` checkout:
+5. You will need `antsibull` for this step. If you haven't installed it yet,
```

Comment:
I would suggest making installing antsibull locally one step ("Install antsibull locally for the next with pip install ...") and then running the actual command can be separate.

Reply:
Step 5 install antsibull and step 6 doing the announcements? Yes, I could change this.

But I'm AFK for the next couple of days. Feel free to do and merge it yourself. Or wait until I'm back to work on it, probably Sunday or Monday.

Reply:
I pushed 0261f3c00e7c4eb06aa20d9e1ab88bf2013683cd for that.

Reply:
@felixfontein LGTM. Thanks!

@gotmax23 What do you think about this?